plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FacetUninstallDelegate.java

```diff
@@ -95,4 +94,53 @@ protected IStatus run(IProgressMonitor monitor) {
     uninstallJob.schedule();
 
   }
+
+  private void removeAppEngineJarsFromMavenProject(IProject project, IProgressMonitor monitor) throws CoreException {
```

Comment:
The corresponding method in `FacetInstallDelegate` is `static`, probably they should both be either `static` or not.